types/block.go

```diff
@@ -1019,7 +1019,7 @@ type Data struct {
 	// have a mechanism to split them out somehow? Probably better to include
 	// them only when necessary (before proposing the block) as messages do not
 	// really need to be processed by tendermint
-	Messages Messages `json:"msgs"`
```

Comment:
[optional] Apologies I can't use Github's suggestion feature b/c the lines above were not modified but I think the comment can be updated and the TODO can be removed b/c it's a question not a concrete action

```golang
// The blobs included in this block.
```

Reply:
https://github.com/celestiaorg/celestia-core/pull/879/commits/e85766db97b5b41899cf46685f714163f4e81007